apps/arc/arc.py

```diff
@@ -20,6 +20,11 @@ def tab_close_wrapper():
         actions.sleep("180ms")
         actions.app.tab_close()
 
+    def please_command(command: str):
```

Comment:
We probably want to call this `command_search` instead. Also the argument should probably be optional.

Reply:
I tried to do this but I'm not sure if I did it right. Please check my work-thank you!

Reply:
I guess there is a discussion if we want to default to an empty string or None.
https://github.com/talonhub/community/blob/940fe694bdb1b5ea752fc2e62154a255f055734e/plugin/talon_draft_window/draft_talon_helpers.py#L224

Also I personally feel that if the argument is optional we should do an if statement an only do sleep and insert if we actually have an argument.
https://github.com/AndreasArvidsson/andreas-talon/blob/114911424dc568d833b865f64f8607ac83275a6d/apps/vscode/vscode_git.py#L63-L67

Reply:
No, not of its optional. You just need to do it away I linked above, but you need to call it from a separate command. `(arg)` doesn't work if `arg` is optional.

Reply:
I went through and added the conditions.  Do you think I should add the separate command? I wasn't sure about that.